Source/Scene/DracoLoader.js

```diff
@@ -72,12 +72,16 @@ define([
         });
         indexBuffer.vertexArrayDestroyable = false;
 
-        return addBufferToModelResources(model, indexBuffer);
+        var bufferViewId = addBufferToModelResources(model, indexBuffer);
+        return {
+            bufferViewId: bufferViewId,
+            numberOfIndices : indexBuffer.numberOfIndices
```

Comment:
Since `numberOfIndices` can get gotten from `bufferViewId` (though indirectly), it may be fine to not store `numberOfIndices` anywhere.

Up to you though, it doesn't matter much to me.

Reply:
I prefer getting all the needed data in one place, then just accessing it later. I did update comments to closely mirror the wording in the [spec](https://github.com/fanzhanggoogle/glTF/blob/cb478d94bf43980ddf057cdb5596dcd9d4282cb2/extensions/Khronos/KHR_draco_mesh_compression/README.md):

> When loading each accessor, you must ignore the bufferView and go to the previously decoded Draco geometry in the primitive to get the data of indices and attributes.

Reply:
Sounds good.